src/test/scala/ie/zalando/fabric/gateway/service/ValidationSpec.scala

```diff
@@ -2,25 +2,61 @@ package ie.zalando.fabric.gateway.service
 
 import cats.data.Validated.{Invalid, Valid}
 import ie.zalando.fabric.gateway.models.HttpModels.ValidationRequest
-import ie.zalando.fabric.gateway.models.ValidationDomain.ResourceDetails
+import ie.zalando.fabric.gateway.models.SynchDomain.{Enabled, Get, Options}
+import ie.zalando.fabric.gateway.models.ValidationDomain.{ResourceDetails, ValidationCorsConfig}
+import io.circe.{Json, JsonObject}
 import org.scalatest.prop.TableDrivenPropertyChecks._
 import org.scalatest.{FlatSpec, Matchers}
 
 class ValidationSpec extends FlatSpec with Matchers {
 
+  private val EmptyJson = Json.fromJsonObject(JsonObject.empty)
+
   "Paths" should "fail the path count validation if they have less than 1 path" in {
-    ResourcePersistenceValidations.validatePathsNonEmpty(ResourceDetails(Nil)) match {
+    ResourcePersistenceValidations.validatePathsNonEmpty(ResourceDetails(Map.empty)) match {
       case Valid(_) => fail("Should not have passed validation")
       case Invalid(nel) =>
         nel.size shouldBe 1
         nel.head.errorMessage shouldBe "There must be at least 1 path defined"
     }
   }
 
+  "Cors Validation" should "pass when cors is enabled and there are no options verbs" in {
+    ResourcePersistenceValidations.validateCorsOptionsRoutes(Enabled, "/resourceDetails", Get) match {
+      case Invalid(_)      => fail("Should not have failed validation")
+      case Valid(pathVerb) => pathVerb shouldBe "/resourceDetails:Get"
+    }
+  }
+
+  "Cors Validation" should "fail when cors is enabled and options endpoints are specified" in {
```

Comment:
You don't need to keep repeating the spec name. You can just use `it` after the first time